example/macos/Podfile.lock

```diff
@@ -1,23 +1,16 @@
 PODS:
   - FlutterMacOS (1.0.0)
-  - gal (1.0.0):
-    - Flutter
-    - FlutterMacOS
 
 DEPENDENCIES:
   - FlutterMacOS (from `Flutter/ephemeral`)
-  - gal (from `Flutter/ephemeral/.symlinks/plugins/gal/darwin`)
 
 EXTERNAL SOURCES:
   FlutterMacOS:
     :path: Flutter/ephemeral
-  gal:
-    :path: Flutter/ephemeral/.symlinks/plugins/gal/darwin
 
 SPEC CHECKSUMS:
   FlutterMacOS: 8f6f14fa908a6fb3fba0cd85dbd81ec4b251fb24
-  gal: 61e868295d28fe67ffa297fae6dacebf56fd53e1
 
 PODFILE CHECKSUM: 23282d8bcd3669427c3011c3d9efa633cc463571
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.16.2
```

Comment:
Updates the CocoaPods version since the previous version doesn't support Xcode 16 and the latest Flutter `main` channel.

Also `Flutter`, `FlutterMacOS`, and `gal` are no longer in here since they support SPM. I didn't remove CocoaPods integration from the example, removing it will make the build faster and no longer need this file but will require contributions to update Flutter SDK to the latest `main` channel, maybe remove it once CocoaPods is no longer supported or SPM become stable (for app packages).

Running the app on latest Flutter `main` channel:

```console
$ flutter run -d macos
All plugins found for macos are Swift Packages, but your project still has
CocoaPods integration. Your project uses a non-standard Podfile and will need to
be migrated to Swift Package Manager manually. Some steps you may need to
complete include:
  * In the macos/ directory run "pod deintegrate"
  * Transition any Pod dependencies to Swift Package equivalents. See
  https://developer.apple.com/documentation/xcode/adding-package-dependencies-to
  -your-app
  * Transition any custom logic
  * Remove the include to "Pods/Target Support
  Files/Pods-Runner/Pods-Runner.debug.xcconfig" in your
  macos/Flutter/Flutter-Debug.xcconfig
  * Remove the include to "Pods/Target Support
  Files/Pods-Runner/Pods-Runner.release.xcconfig" in your
  macos/Flutter/Flutter-Release.xcconfig

Removing CocoaPods integration will improve the project's build time.
```